modules/manage/pages/manage-backup-and-restore/manage-backup-and-restore.adoc

```diff
@@ -0,0 +1,799 @@
+The name must be unique across the cluster, can only use the characters `[`, `]`, `A` to `Z`, `a` to `z`, `&#95;` and `-`; and must not start with either `&#95;` or `-`.
++
+The *Storage Location* can be specified as *Filesystem* (the default) or *Cloud*.
+For the current example, *Filesystem* will be used.
+Note that if *Cloud* is selected, allowing AWS S3 storage to be used, the dialog expands, and displays additional options: these are described below, in xref:manage:manage-backup-and-restore/manage-backup-and-restore.adoc#use-cloud-storage[Use Cloud Storage].
++
+In the *Cbbackupmgr Repository Path*, enter the path of the repository to be imported.
+If necessary, locate on disk the folder within which the repository resides:
++
+image:manage-backup-restore/repoOnDisk1.png[,110,align=left]
++
+Locate the repository within its parent folder, and make a copy of the repository name:
++
+image:manage-backup-restore/repoOnDisk2.png[,200,align=left]
++
+Then, specify the repository name as part of the full file path.
+The *Import Repository* dialog now appears as follows:
++
+image:manage-backup-restore/importRepositoryComplete.png[,420,align=left]
+
+. Left-click on the *Import* button, at the lower right of the dialog.
+The dialog disappears, and the repository is imported.
+
+. Check the results of the import operation.
+Access the *Repositories* screen, and observe the *Imported Repositories* panel:
++
+image:manage-backup-restore/importedRepositoriesPanel.png[,720,align=left]
++
+For further details, left-click on the row to make it expand vertically:
++
+image:manage-backup-restore/importedRepositoriesPanelExpanded.png[,720,align=left]
+
+[#inspect-plans]
+== Inspect Plans
+
+All plans created for the Backup Service can be reviewed, by left-clicking on the *Plans* tab, on the upper, horizontal navigation bar:
+
+image:manage-backup-restore/plansTab.png[,260,align=left]
+
+This displays the *Plans* screen:
+
+image:manage-backup-restore/plansScreen.png[,720,align=left]
+
+The plan displays, in the *name* column, the name of every current plan; under the *# Used by* column, an integer that represents the number of repositories maintained by means of the plan identified on the current row; and under *services*, the Couchbase Services whose data is handled by the plan.
+
+To inspect a plan in detail, left-click on its row, to make the row expand vertically.
+For example:
+
+image:manage-backup-restore/exportAndMergePlanExpanded.png[,720,align=left]
+
+The displayed details indicate that the *BackupAndMerge* plan contains two tasks: the *BackupTask* performs a task of type *BACKUP* every 2 minutes; while the *MergeTask* performs a task of type *MERGE* every 10 minutes, merging all backups.
+
+At the right-hand side of each row, a garbage-can icon is displayed.
+To delete a plan, left-click on the icon on the row of the plan to be deleted: the plan is deleted immediately, and no longer appears on the *Plans* screen.
```

Comment:
It may be worth mentioning that a plan that is being used cannot be deleted until the repositories using it are archived or deleted

Reply:
I've added: (Note that a plan cannot be deleted until all repositories using it are archived or themselves deleted.)

Reply:
At https://github.com/couchbase/docs-server/pull/1596